config/schema/og.schema.yml

```diff
@@ -1,3 +1,22 @@
+field.storage_settings.og_membership_reference:
+  type: mapping
+  label: 'Organic Groupds reference field storage settings'
+  mapping:
+    target_type:
+      type: string
+      label: 'Type of entity to reference'
+
+field.field_settings.og_membership_reference:
+  type: mapping
+  label: 'Organic Groupds reference field settings'
+  mapping:
+    handler:
+      type: string
+      label: 'Reference method'
+    handler_settings:
+      type: entity_reference_selection.[%parent.handler]
```

Comment:
This is for a dynamic type in the schema, see https://www.drupal.org/node/1905070 there is a cheat sheet PDF you can get on there which is actually pretty useful. This basically allows this configuration to stay the same, but change if the selection plugin were to change. We could just not have this for now, but seems like we might as well just leave it in. Makes switching it out easier/possible.